### DISCUSSION TITLE
Jamulus 3.8.2 Beta 1 has been released

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#2316] [Announcements]
**THIS IS A BETA VERSION!**

Expect the possibility of bugs, and please tell us about them.

We have just released version 3.8.2beta1 of Jamulus, and would be happy for you to download, compile and test out this version.

You can view the Release Notes and download versions for Windows, Mac, Linux (Debian/Ubuntu) and Android (experimental) at https://github.com/jamulussoftware/jamulus/releases/tag/r3_8_2beta1. You can also download the source code as a zip or tarball for compiling, or you can check out the code using git.

If you find any issues with it, please raise an Issue with as many details as possible.

**Comment by hoffie** [comment]
Thanks to @emlynmac the Mac binary has just been replaced by the signed version.

**Comment by BentWrench** [comment]
Installed on my server Rob's Pi if anyone on the west coast wants to try it out.

**Reply by hoffie** [reply]
Thanks for helping with testing by providing that server :)

**Comment by hoffie** [comment]
Closing this thread in favor of #2389. If you've encountered any issues with this beta, please check the newly released rc1. If it still shows issues, please raise an [Issue](https://github.com/jamulussoftware/jamulus/issues/new/choose).
